### DISCUSSION TITLE
CZ language pack

### DISCUSSION THREAD

**Discussion by japanel** [OpenI6X/opentx#395] [General]
Hello, I would like to ask about possibility of CZ sound pack. Is it possible? Maybe it is compatible with some OTX sound pack? Thanks

**Comment by ajjjjjjjj** [comment]
Yes it is with some adjustments, see: https://github.com/OpenI6X/opentx/pull/377

**Reply by japanel** [reply]
Hi, I saw it, but which adjustment you mean pls?

**Reply by ajjjjjjjj** [reply]
You need to take a look at CZ differences and do the same what was done for EN. Sorry for late response, I don't have much time lately.

**Reply by japanel** [reply]
I have it done. 👍
